doc/Makefile.am

```diff
@@ -4,7 +4,8 @@
 
 SUBDIRS = man specs sag adg mwg
 
-CLEANFILES = *~ custom-html.xsl custom-man.xsl
+CLEANFILES = *~
+DISTCLEANFILES = custom-html.xsl custom-man.xsl
```

Comment:
Fixes b285afe0ea45f391b819b9f61316827c59ba6c56 which was not included in a release. Strongly agree and thanks a lot for spotting and describing the consequences!